src/content/articles/alab.md

```diff
@@ -88,30 +88,47 @@ downloadSection: {
   # The download links and button setup for the download table.
   downloads: [{
     buttonText: "DOWNLOAD",
-    downloadUrl: "https://dpel-assets.aswf.io/usd-alab/alab-v2.1.0.zip",
+    downloadUrl: "https://dpel-assets.aswf.io/usd-alab/alab-v2.2.0.zip",
     size: "9.3 GB",
-    description: "USD Scene - v.2.1.0",
+    description: "",
+    descriptionBold: "Asset Structure | ",
+    extraDescription: "Main asset structure for the ALab showcasing how all assets relate to each other through USD composition arcs. This is purely the USD structure linking all files, and does not include any geometry, shaders or lights.",
     type: "primary"
   },
   {
     buttonText: "DOWNLOAD",
-    downloadUrl: "https://aswf-dpel-assets.s3.amazonaws.com/usd-alab/alab-textures.v2.1.0.zip",
+    downloadUrl: "https://dpel-assets.aswf.io/usd-alab/alab-v2.2.0.zip",
```

Comment:
This URL will be https://dpel-assets.aswf.io/usd-alab/alab-techvars.v2.2.0.zip once it is migrated to the correct bucket.